sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/aio/operations/_container_registry_blob_operations.py

```diff
@@ -1,17 +1,17 @@
+# pylint: disable=too-many-lines
 # coding=utf-8
 # --------------------------------------------------------------------------
-# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.7.4, generator: @autorest/python@5.12.4)
+# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.1.3, generator: {generator})
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
-import functools
-from typing import Any, Callable, Dict, Generic, IO, Optional, TypeVar
-import warnings
+from typing import Any, Callable, Dict, IO, Optional, TypeVar
 
 from azure.core.exceptions import ClientAuthenticationError, HttpResponseError, ResourceExistsError, ResourceNotFoundError, map_error
 from azure.core.pipeline import PipelineResponse
 from azure.core.pipeline.transport import AsyncHttpResponse
 from azure.core.rest import HttpRequest
 from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.utils import case_insensitive_dict
```

Comment:
`case_insensitive_dict` was only [made public](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/CHANGELOG.md#1230-2022-03-03) in `azure-core` 1.23.0, so the version in [setup.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/containerregistry/azure-containerregistry/setup.py#L58) will need to be updated -- as well as the [override in `shared_requirements.txt`](https://github.com/Azure/azure-sdk-for-python/blob/main/shared_requirements.txt#L136)